### DISCUSSION TITLE
Disable `cursorline`

### DISCUSSION THREAD

**Discussion by stabldev** [nvim-neo-tree/neo-tree.nvim#1567] [Q&A]
I searched everywhere, yet couldn't find a way to disable cursorline for neotree buffer.

![image](https://github.com/user-attachments/assets/089d5495-1111-449d-b8db-a7087d285b18)

is there anything that i can do in this?
```lua
return {
  "nvim-neo-tree/neo-tree.nvim",
  opts = {},
}
```

**Comment by miversen33** [comment]
I don't know that we have a way to disable the cursorline outright, but you can customize it with the `NeoTreeCursorLine` Highlight group.

**Reply by stabldev** [reply]
yes, I needed to customize `custom_highlights` on theme itself, like:
```lua
return {
  "catppuccin/nvim",
  lazy = true,
  name = "catppuccin",
  opts = {},
  config = function()
    require("catppuccin").setup({
      term_colors = true,
      transparent_background = true,

      custom_highlights = function(colors)
        local u = require("catppuccin.utils.colors")
        return {
          CursorLineNr = { bg = u.blend(colors.overlay0, colors.base, 0.75), style = { "bold" } },
          CursorLine = { bg = u.blend(colors.overlay0, colors.base, 0.45) },
          LspReferenceText = { bg = colors.surface2 },
          LspReferenceWrite = { bg = colors.surface2 },
          LspReferenceRead = { bg = colors.surface2 },
        }
      end,
    })
  end,
}

end,

```
well this doesn't disable it like `:set nocursorline`, but fine to me
might help for others as well.
